packages/opentelemetry-test-utils/testUtils.ts

```diff
@@ -15,13 +15,16 @@
  */
 
 import * as childProcess from 'child_process';
+// eslint-disable-next-line node/no-unpublished-import
```

Comment:
is this really needed ? and then line 21 and 27 ?

Reply:
I followed what was suggested on this [doc](https://docs.google.com/document/d/1bxrwuxEP6acxRXGe7hT5THJLjYXm3Bt924jZ72C_82s/edit?usp=sharing). As Daniel mentioned, this could also be resolved by moving these modules from devDependencies to dependencies. What do you think?

Reply:
This is a dependency of the package so it should be a real dependency.